### PR TITLE
Custom converison source

### PR DESCRIPTION
**Pull request by agnessnowplow**

### CODE REVIEW

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md [hunk 1]
```diff
@@ -125,6 +125,30 @@ You will also need a table where the conversion events are stored. If you use th
 To fully finish the config you could overwrite the `snowplow__conversion_clause` variable in your project.yml, in case you would want to filter on specific types of conversions. You should also set the `snowplow__conversion_stitching` variable to true, if you have stitched_user_id in your conversions table.
 :::
 
+In case you store your conversion events elsewhere in the data warehouse (e.g. because those are transactional data that you decide you do not want to process through Snowplow), you can mutate that source table into the [format of the conversions table](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/conversions/#2-enabling_conversions_in_the_conversions_module) generated by the unified package, store it as a sql view, and refer to that in the variable `snowplow__conversion_source` (in the format of `schema.view_name`).
```
Comment (rlh1994) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md:128]: Needs to be an index.md link please

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md [hunk 2]
```diff
@@ -125,6 +125,30 @@ You will also need a table where the conversion events are stored. If you use th
...
 :::
 
+In case you store your conversion events elsewhere in the data warehouse (e.g. because those are transactional data that you decide you do not want to process through Snowplow), you can mutate that source table into the [format of the conversions table](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/conversions/#2-enabling_conversions_in_the_conversions_module) generated by the unified package, store it as a sql view, and refer to that in the variable `snowplow__conversion_source` (in the format of `schema.view_name`). 
+
+You could also union multiple sources (e.g. one transactional event data for automatic conversion events not dictated by user actions and the the rest, ending up in the unified_conversions table). 
+
+Please bear ind mind, if you would like to calculate the attribution for conversion events not processed by Snowplow, you would need to make sure these events contain a user_id type field which the package can rely on to find previous website visits for it to be able to attribute them to these conversions. Otherwise your overall revenue will be more than the total attributed revenue.
```
Comment (rlh1994) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md:132]: Ind

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md [hunk 3]
```diff
@@ -125,6 +125,30 @@ You will also need a table where the conversion events are stored. If you use th
...
+
+<details>
+    <summary>Conversion formatting requirements</summary>
+
+  Fields to use (including the name) when creating the view:
+
+- `event_id`: can be dummy UUID, needs to be unique though
```
Comment (rlh1994) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md:139]: Maybe just label which of these are required or optional more clearly?

**Review by rlh1994 — CHANGES_REQUESTED** [review]

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md (hunk 3 above)
Comment (agnessnowplow) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md:139]: it is hard cause it is optional only if they do not do union, did something that kinda explains both

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md (hunk 1 above)
Comment (agnessnowplow) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md]: can't get numbered ones to actually take you to that section but a quick scroll and they are there anyway

**Review by rlh1994 — APPROVED** [review]

**Pull request merged**